lib/atom-music-view.js

```diff
@@ -238,7 +238,7 @@ class AtomMusicView {
 		}
 	}
 
-	filesBrowsed(e) {
+	onFilesChanged(e) {
```

Comment:
@UziTech `onFilesBrowsed` might be more clear.

Reply:
This is called when the "files" input changed. `on={{change: this.onFilesChanged}} `

I don't think "browsed" would make it any more clear.

Reply:
Yes, `files` changes. 
But the described process is browsing for files to add.
`..Changed` might sound a little bit like observing files for changes, i think.

Reply:
Maybe `onPlaylistChange`? I feel like the names of these functions aren't really important enough to need to be changed. This function, for example, is only used in one place. It is not difficult for anyone to know what it does no matter what name is used.

Reply:
@sdinesh86 What do you think?

Reply:
@UziTech I will change it to `onFilesAdded`.